src/server/utils/push.util.js

```diff
@@ -0,0 +1,66 @@
+import FCM from 'fcm-push';
+import mongoose from 'mongoose';
+
+const serverKey = 'AAAATC_Jnfw:APA91bE6u80DCTiBgZdI8Fkuc6RHzblvKfWw4gTmFc1aT7Ud1eyGtaSplg8wa6UwOO4' +
+  'CxHa5Hf0orYoh9c8DKxiKrH5DB5QToCiPbwOPwP-eZgLbdvN3s4FH8an4Q0fBk7itK8tEcIQI6dIYnct_j1O2ExGEN9FChg';
+const fcm = new FCM(serverKey);
+
+const User = mongoose.model('user');
+
+const NOTIFICATION_CONFIG = {
+  sound: 'default',
+  vibrate: 700,
+};
+
+const NOTIFICATION_TYPE = {
+  REQUEST: {
+    title: 'New Request',
+    bodyParam: ' has requested a connection to you.',
+  },
+  CONNECTION: {
+    title: 'New Connection',
+    bodyParam: ' and you are connected now.',
+  },
+  MESSAGE: {
+    title: 'New Message',
+    bodyParam: 'You’ve got new message.',
+  },
+};
+
+export function sendPush(senderId, receiverId, notificationType, senderName, message) {
```

Comment:
I think we can make this function with only 3 params.
receiverId, notificationType, body.